src/proportional-bar/ProportionalBar.js

```
@@ -122,4 +122,9 @@ ProportionalBar.propTypes = {
     })
   ).isRequired,
   title: PropTypes.string.isRequired,
+  showLegend: PropTypes.bool,
+};
+
+ProportionalBar.defaultProps = {
+  showLegend: false,
```

I would default this to `true`, we should only be hiding it in special cases